bin/run-jetpack-command.sh

```diff
@@ -32,16 +32,14 @@ pnpm_version=$(npx semver -c "$listed_pnpm_version")
 # More information in: https://github.com/wordpress-mobile/gutenberg-mobile/issues/5688
 sed -i.bak 's/^engine-strict = true/engine-strict = false/' .npmrc
 
-# Install dependecies of Jetpack plugin
-pushd projects/plugins/jetpack
-
 # npx might prompt to install pnpm at the requested version. Let's just agree and carry on.
-( yes || true ) | npx --cache /tmp/empty-cache pnpm@"$pnpm_version" install --ignore-scripts
-
-popd
+( yes || true ) | npx --cache /tmp/empty-cache pnpm@"$pnpm_version" $1
 
 # Revert `engine-strict` parameter back
 sed -i.bak 's/^engine-strict = false/engine-strict = true/' .npmrc
 rm .npmrc.bak
 
 popd
+
+# Revert to Gutenberg node version
+nvm use
```

Comment:
I noticed when running this script that after the execution, I was on a different node version. Hence, I added that `nvm` changes the node version to the one recommended in Gutenberg/Gutenberg Mobile.